src/main/java/mate/academy/model/Movie.java

```diff
@@ -7,7 +7,7 @@
 import jakarta.persistence.Table;
 
 @Entity
-@Table(name = "movies")
+@Table(name = "movie")
```

Comment:
why rename it to singular form? database tables are usually plural as of the style convention, same for other entities